src/test/java/org/folio/innreach/domain/listener/KafkaCirculationEventListenerApiTest.java

```diff
@@ -113,14 +119,20 @@
   private InnReachExternalService innReachExternalService;
 
   @MockBean
-  private ItemService itemService;
+  private HoldingsService holdingsService;
```

Comment:
ideally services should not be mocked, because they are important part of business code
instead mock a feign client that talks to holding module

Reply:
Fixed